tests/assert.js

```diff
@@ -0,0 +1,49 @@
+export function assert(actual, expected, message) {
+    if (arguments.length == 1)
+        expected = true;
+
+    if (typeof actual === typeof expected) {
+        if (actual === expected) {
+            if (actual !== 0 || (1 / actual) === (1 / expected))
+                return;
+        }
+        if (typeof actual === 'number') {
+            if (isNaN(actual) && isNaN(expected))
+                return true;
```

Comment:
```suggestion
                return;
```
All other returns are plain returns.